kie-server-parent/kie-server-controller/kie-server-controller-openshift/src/main/java/org/kie/server/controller/openshift/storage/ServerTemplateConverter.java

```diff
@@ -109,34 +123,37 @@ public static ServerTemplate fromState(KieServerState state) {
 
         List<String> capabilities = Stream.of(Capability.values()).map(Capability::name).collect(Collectors.toList());
         KieServerConfig kcfg = state.getConfiguration();
-        if (kcfg.getConfigItem(KieServerConstants.KIE_JBPM_SERVER_EXT_DISABLED) != null) {
+        if (kcfg.getConfigItem(KIE_JBPM_SERVER_EXT_DISABLED) != null) {
             capabilities.remove(Capability.PROCESS.name());
         }
-        if (kcfg.getConfigItem(KieServerConstants.KIE_DROOLS_SERVER_EXT_DISABLED) != null) {
+        if (kcfg.getConfigItem(KIE_DROOLS_SERVER_EXT_DISABLED) != null) {
             capabilities.remove(Capability.RULE.name());
         }
-        if (kcfg.getConfigItem(KieServerConstants.KIE_OPTAPLANNER_SERVER_EXT_DISABLED) != null) {
+        if (kcfg.getConfigItem(KIE_OPTAPLANNER_SERVER_EXT_DISABLED) != null) {
             capabilities.remove(Capability.PLANNING.name());
         }
 
         template.setCapabilities(capabilities);
-        template.setMode(KieServerMode.valueOf(mode));
+        //https://issues.jboss.org/projects/RHPAM/issues/RHPAM-1975
+        template.setMode(KieServerMode.valueOf(mode.toUpperCase()));
 
         return template;
     }
 
     public static KieServerState toState(ServerTemplate template) {
         String id = template.getId();
-        KieServerMode mode = template.getMode() == null ? KieServerMode.DEVELOPMENT : template.getMode();
+        String name = template.getName();
+        KieServerMode mode = template.getMode() == null ? KieServerMode.PRODUCTION : template.getMode();
```

Comment:
Any reason why to switch default mode to PRODUCTION?

Reply:
Wasn't clear what is the default value. Checked both the Cloud image scripts, which does NOT set the default, and KIE server implementation, and found this:

https://github.com/rhtevan/droolsjbpm-integration/blob/dd66cf9ebd5c975b3a2788819653f00c5ac970ad/kie-server-parent/kie-server-controller/kie-server-controller-openshift/src/main/java/org/kie/server/controller/openshift/storage/ServerTemplateConverter.java#L68

To align with KIE server implementation, changed it back to DEVELOPMENT.